dashboard/app/models/script.rb

```diff
@@ -380,7 +380,11 @@ def self.beta?(name)
 
   # TODO(asher): Rename this method to k1?, removing the need to disable lint.
   def is_k1?  # rubocop:disable PredicateName
-    name == 'course1'
+    [
+      Script::COURSEA_DRAFT_NAME,
+      Script::COURSEB_DRAFT_NAME,
+      Script::COURSE1_NAME
+    ].include?(name)
```

Comment:
Is there any reason k1 isn't a property on individual scripts, instead of doing it this way?